docs/glossary.md

```diff
@@ -0,0 +1,7 @@
+# Glossary
+
+Glossary of all experiment attributes with short definitions.
+
+* **Fractional factorial**
+  : A design that is the fraction of a full-factorial design.
+  A full-factorial $2^{k}$ contains all combinations of the levels of $k$ factors.
```

Comment:
github's markdown does not understand late syntax, that's a pity. There are some solutions out there but nothing we can directly use. We can talk about it tomorrow but for the moment I find this OK